src/main/java/com/health/BMI_check/controller/BodyDataController.java

```diff
@@ -0,0 +1,32 @@
+package com.health.BMI_check.controller;
+
+import com.health.BMI_check.entity.BodyData;
+import com.health.BMI_check.service.BmiService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+
+@RestController
+public class BodyDataController {
+    private final BmiService bmiService;
+
+    public BodyDataController(BmiService bmiService) {
+        this.bmiService = bmiService;
+    }
+
+    /**
+     * Create処理実装 ユーザー情報に関してHTTPリクエストを受け付けるController
+     */
+    @PostMapping("/BMIs")
```

Comment:
今だとコントローラの説明になっていますが、ここに記述するべきコメントはメソッドの説明ですね。
```suggestion

    /**
     * Create処理実装 ユーザー登録に関してHTTPリクエストを受け付ける
     */
    @PostMapping("/BMIs")
```

Reply:
コメントの位置によって、説明の内容が変わるのですね。
- クラスを定義する前: クラスがどんな役割を持つか
- メソッドの前: 下に書いてあるメソッドがどんな働きをするか

という認識で大丈夫でしょうか。

suggestion の通りにコメントの部分を修正してみました！[20e0cd2](https://github.com/teddypr/BMI-check/pull/11/commits/20e0cd2cba666a361f74966884ef5fceefedbc76)

Reply:
その認識であってます！